### ISSUE TITLE
Tabs missing accessibility functionality

### ISSUE THREAD

**Issue by mohammedzamakhan** [reach/reach-ui#125]
Thanks for the great accessible tabs component.

The only missing functionality from the w3c tabs accessibility are these three keys

**Home** | Moves focus to the first tab and activates it.
**End** | Moves focus to the last tab and activates it.
**Delete** | When focus is on the Joke tab, removes the tab from the tab list and places focus on the previous tab.
https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

I do not know if delete is important to support, but home and end are important for sure. Other UI Libraries that focuses on accessibility do support them like https://react.collab-ui.com/layout/tabs and https://material.angular.io/components/tabs/overview 

Collab UI also supports additional functionality, where the user can type the first letter from the title and react to that particular tab. Very good for keyboard users if there are like more than 10 different tabs

**Comment by rubenmoya** [comment]
Both `Home` and `End` are already implemented, do we want to implement `Delete` to? Maybe adding an `onDelete` prop, what do you think?

**Comment by chancestrickland** [comment]
I'm not sure we'll support `delete` by default, though it may be worth creating an example on the docs site. Here's a quick and dirty example on code sandbox that can probably be better, but I'll use it for reference if I get around to making this for the docs. https://codesandbox.io/s/deleteable-tabs-27xds
